### PR TITLE
Fix icon base64data and code block formatting in CSV files

### PR DESCRIPTION
**Pull request by michael-valdron**
**Please specify the area for this PR**

registry operator

**What does does this PR do / why we need it**:

It is reported in devfile/api#1354 that the icon is missing and code blocks are not formatting correctly under the embedded OperatorHub catalog in OpenShift. These changes mirror the formatting of the strings used in other bundle entries with expected results which should correct these problems.

Testing these changes under the [OperatorHub.io preview tool](https://operatorhub.io/preview) does not break the currently working page.

**Note**: These changes require a patch version to be cut before devfile/api#1354 can be resolved.

**Which issue(s) this PR fixes**:

Fixes #?

devfile/api#1354

**PR acceptance criteria**:

- [X] Test Coverage 
    - Are your changes sufficiently tested, and are any applicable test cases added or updated to cover your changes?
- [x] Gosec scans 
  - Fix all MEDIUM and higher findings and/or annotate a finding per gosec instructions: https://github.com/securego/gosec#annotating-code to address why a finding is not a security issue

Documentation
- [ ] Does the registry operator documentation need to be updated with your changes?

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

**Comment by michael-valdron** [comment]
> Documentation
>
> Does the registry operator documentation need to be updated with your changes?

N/A

File: bundle/manifests/registry-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -615,7 +448,7 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
-                image: quay.io/devfile/registry-operator:v0.1.3
+                image: quay.io/devfile/registry-operator:next
```
Comment (thepetk) [inline bundle/manifests/registry-operator.clusterserviceversion.yaml:451]: Out of curiosity, I see the `spec.version` field inside the `config/manifests/bases/registry-operator.clusterserviceversion.yaml`. Should we also update this field as `next`?

**Review by thepetk — COMMENTED** [review]
In general lgtm only left a minor comment. If we are ok I'll approve!

File: bundle/manifests/registry-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (michael-valdron) [reply bundle/manifests/registry-operator.clusterserviceversion.yaml:451]: This field would be updated to the new version cut `v0.1.4` in a separate release PR, we can leave it `next` under the main branch.

**Review by thepetk — APPROVED** [review]
/lgtm

**Pull request merged**